apps/src/templates/studioHomepages/teacherHomepageV2/SectionOptionsDropdown.tsx

```diff
@@ -36,31 +44,97 @@ const onLoginCardsClick = (navigate: NavigateFunction, sectionId: number) => {
   );
 };
 
+const onCertificatesClick = (section: Section) => {
+  HttpClient.fetchJson<Student[]>(
+    `/dashboardapi/sections/${section.id}/students`
+  )
+    .then(response => response.value)
+    .then((value: Student[]) => {
+      const students = value.map(student => student.name);
+      const courseVersionName: string = section.courseVersionName || '';
+      const urlParams = new URLSearchParams([
+        ['course', btoa(courseVersionName)],
+      ]);
+      students.map(student => urlParams.append('names[]', student));
+      window.location.href = `/certificates/batch?${urlParams.toString()}`;
+    });
+};
+
+const onArchiveClick = (dispatch: AppDispatch, section: Section) => {
```

Comment:
You can't archive or delete sections that you didn't create (e.g. if you were added as a coteacher). Should we add some logic (in a future PR?) to remove these options in those cases?

Reply:
In [the original co-teacher spec](https://docs.google.com/document/d/1zyODkc7VXsAb17W3jefNM9JNkQVZ8nU2q5C9Xlp7cZs/edit?usp=drivesdk), co-teachers have all the same permissions on the section as the section owner including archiving and deleting, and I don't think we've changed that since the original implementation...so I don't think we need to have logic around this :)